app/views/welcome/_footer.html.erb

```diff
@@ -0,0 +1,26 @@
+<footer class="page-footer grey darken-4">
+  <div class="container">
+    <div class="row">
+      <div class="col l6 s12">
+        <h5 class="white-text">Footer Content</h5>
+        <p class="grey-text text-lighten-4">This is where the motto of finden will be</p>
```

Comment:
We're using rails internationalization now, please move this text to a locale file.

Reply:
first i want to finish this commit and then i can add the i18n library